### PR TITLE
Add Kraum, Violent Cacophony OTJ

### PR DESCRIPTION
**Pull request by loud1990**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/kraum_violent_cacophony [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Kraum, Violent Cacophony
+ManaCost:2 U R
+Types:Legendary Creature Zombie Horror
+PT:2/3
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, put a +1/+1 counter on Kraum, Violent Cacophony and draw a card
+SVar:TrigDraw:DB$ Draw | NumCards$ 1 | SubAbility$ TrigPutCounter
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
+Oracle:Flying \nWhenever you cast your second spell each turn, put a +1/+1 counter on Kraum, Violent Cacophony and draw a card
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kraum_violent_cacophony:9]: remove trailing space and add .

File: forge-gui/res/cardsfolder/upcoming/kraum_violent_cacophony [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Kraum, Violent Cacophony
+ManaCost:2 U R
+Types:Legendary Creature Zombie Horror
+PT:2/3
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, put a +1/+1 counter on Kraum, Violent Cacophony and draw a card
+SVar:TrigDraw:DB$ Draw | NumCards$ 1 | SubAbility$ TrigPutCounter
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kraum_violent_cacophony:7]: switch effect order according to the printed text

File: forge-gui/res/cardsfolder/upcoming/kraum_violent_cacophony [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Kraum, Violent Cacophony
+ManaCost:2 U R
+Types:Legendary Creature Zombie Horror
+PT:2/3
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, put a +1/+1 counter on Kraum, Violent Cacophony and draw a card
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/kraum_violent_cacophony:6]: `.` at end of TrigDesc

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/kraum_violent_cacophony:6]: also use CARDNAME in TrigDesc

**Review by tool4ever — APPROVED** [review]
thanks

**Pull request merged**